src/Default/Middleware/Authorize.php

```diff
@@ -2,19 +2,17 @@
 
 namespace Sowl\JsonApi\Default\Middleware;
 
+use Illuminate\Http\Request as HttpRequest;
```

Comment:
In all the library I use logic in the "use" block that first of all we import internal classes and then imports of the external vendor classes.

I don't remember from where I got this standard can't find the rule.